src/components/Account/Account.tsx

```diff
@@ -237,7 +250,13 @@ const Account = ({ panelClassName, panelHeaderClassName, canUpdateEmail = true }
           }),
           formSection({
             label: t('account.security'),
-            editButton: <Button label={t('account.edit_password')} type="button" onClick={() => (customer ? editPasswordClickHandler() : null)} />,
+            editButton: (
+              <Button
+                label={t(shouldAddPassword ? 'account.add_password' : 'account.edit_password')}
```

Comment:
To make the translations extractable:

```suggestion
                label={shouldAddPassword ? t('account.add_password') : t('account.edit_password')}
```